From: "Matteo F. Vescovi" <mfv@debian.org>
Date: Tue, 3 Feb 2015 16:55:24 +0100
Subject: Drop_PHONY_on_pc_files

---
 Makefile | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/Makefile b/Makefile
index 6a9803c..b609eb7 100644
--- a/Makefile
+++ b/Makefile
@@ -127,10 +127,6 @@ tmp/$(SYSTEM)/default/shared/glew.o: src/glew.c include/GL/glew.h include/GL/wgl
 	@mkdir -p $(dir $@)
 	$(CC) -DGLEW_NO_GLU -DGLEW_BUILD $(CFLAGS) $(CFLAGS.SO) -o $@ -c $<
 
-# Force re-write of glew.pc, GLEW_DEST can vary
-
-.PHONY: glew.pc
-
 glew.pc: glew.pc.in
 	sed \
 		-e "s|@prefix@|$(GLEW_PREFIX)|g" \
@@ -168,10 +164,6 @@ tmp/$(SYSTEM)/mx/shared/glew.o: src/glew.c include/GL/glew.h include/GL/wglew.h
 	@mkdir -p $(dir $@)
 	$(CC) -DGLEW_NO_GLU -DGLEW_MX -DGLEW_BUILD $(CFLAGS) $(CFLAGS.SO) -o $@ -c $<
 
-# Force re-write of glewmx.pc, GLEW_DEST can vary
-
-.PHONY: glewmx.pc
-
 glewmx.pc: glew.pc.in
 	sed \
 		-e "s|@prefix@|$(GLEW_PREFIX)|g" \
